roles/dns-slave/files/named.conf.options

```diff
@@ -0,0 +1,13 @@
+options {
+	directory "/var/cache/bind";
+
+	forwarders {
+		152.78.110.110;
```

Comment:
What are these? Use vars and then add comments?

Reply:
We forward all non-sown queries out to the standard set of soton resolvers

Reply:
Yeah, so have a var for forwarders? Maybe idk

Reply:
I just added a comment explaining what they are in dd143b56f812cc762364c3c71da0d394310e877d - it'll be identical config between servers, so not worth templating out